packages/assets-controllers/src/TokensController.test.ts

```diff
@@ -1457,7 +1457,9 @@ describe('TokensController', () => {
             }),
           },
         },
-        async ({ controller }) => {
+        async ({ controller, changeNetwork }) => {
+          changeNetwork({ selectedNetworkClientId: InfuraNetworkType.goerli });
```

Comment:
Should we update the test name, or add a comment here on why we are changing a network?